pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -1,5 +1,13 @@
 { lib, config, stdenv }:
 
+let
+  check-meta = import ./check-meta.nix {
+        inherit lib config;
```

Comment:
Make sure to use the 2 space indentation.

Reply:
Done, thanks :)